test/SlowTests/MailingList/RavenArrayQueryInputTests.cs

```diff
@@ -1,59 +0,0 @@
-using System;
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.MailingList
-{
-    public class RavenArrayQueryInputTests : RavenTestBase
-    {
-        private class Document
-        {
-            public string Id { get; set; }
-        }
-
-        private class TransformResult
-        {
-            public string Id { get; set; }
-            public int[] Array { get; set; }
-        }
-
-        private class TestTransformer : AbstractTransformerCreationTask<Document>
-        {
-            public TestTransformer()
-            {
-                TransformResults = documents => from d in documents
-                                                let index = Array.IndexOf(Parameter("array").Value<string>().Split(',', StringSplitOptions.None), d.Id)
-                                                orderby index
```

Comment:
You don't. We won't allow to send more results from the function.
You can send an object, what you do with the object is completely up to you